#requesting the change in service
string inp_message
---
#giving the input to the called service
string out_message

